server/routerlicious/packages/services-shared/src/restLessServer.ts

```diff
@@ -99,8 +108,9 @@ export class RestLessServer {
     }
 
     private async parseStreamRequestFormBody(request: IncomingMessageEx): Promise<void> {
+        const formidable = (await inclusion("formidable")).default;
         return new Promise<void>((resolve, reject) => {
-            const form = formidable({ multiples: true });
+            const form = formidable();
```

Comment:
You mentioned in the other closed PR that `multiples` support was removed. How does that affect us?

Reply:
Formidable now always returns an array, where it used to be dependent on the number of form fields with that field name (1 field name would return `string`, multiple fieldnames would return `array`). We made use of this expecting multiple Header fields and handling the array possibility. However, for body and method, we expected 1, so a `string`. You can see the type changes above in the code (where you commented already). We still need to handle `string` because if the body is parsed by `bodyparser` instead, it will be a `string | string[]`